Index: .idea/vcs.xml
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- .idea/vcs.xml	(revision )
+++ .idea/vcs.xml	(revision )
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$" vcs="Git" />
+  </component>
+</project>
\ No newline at end of file
Index: modeller.py
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- modeller.py	(date 1507664225000)
+++ modeller.py	(revision )
@@ -1,1 +1,31 @@
-pass
+import numpy as np
+
+def chooser(buf_vip, buf_oth, t_vip, t_oth):
+    pass
+
+
+lambda_vip = 1
+t_vip = 0
+lambda_oth = 4
+t_oth = 0
+
+t_proc = 1
+
+buf_vip = []
+buf_oth = []
+
+query_num = 10000
+# Расписание получения заявок
+vip_times = np.random.exponential(lambda_vip, [1, query_num])
+oth_times = np.random.exponential(lambda_oth, [1, query_num])
+print(str(np.mean(vip_times)))
+print(str(np.mean(oth_times)))
+vip_requests = [np.sum(vip_times[:i]) for i in range(query_num)]
+oth_requests = [np.sum(oth_times[:i]) for i in range(query_num)]
+
+print(str(np.mean(vip_requests)))
+print(str(np.mean(oth_requests)))
+print(vip_requests[10])
+
+
+
